src/hooks/plan_tree_walker.c

```diff
@@ -55,7 +57,7 @@ bool plan_tree_walker(Plan *plan, bool (*walker_func)(Node *plan, void *context)
         // Join nodes
         case T_Join:
             Join *join = (Join *)plan;
-            if(base_plan_walker(&(join->plan), walker_func, context)) return true;
+            if(base_plan_walker((Plan *)&(join->plan), walker_func, context)) return true;
```

Comment:
I don't think this or the one below should be necessary, join->plan is always a plan.

Reply:
You are right. removed it.